framework/xds_k8s_testcase.py

```diff
@@ -560,6 +562,20 @@ def _assertRpcsEventuallyGoToGivenServers(
                 f"Unexpected server {server_hostname} received RPCs",
             )
 
+    def assertEDSConfigExists(self, config: ClientConfig):
+        seen = set()
+        want = frozenset(["endpoint_config"])
+        for generic_xds_config in config.generic_xds_configs:
+            if re.search(
+                r"\.ClusterLoadAssignment$", generic_xds_config.type_url
+            ):
+                seen.add("endpoint_config")
+        logger.debug(
+            "Received xDS config dump: %s",
+            json_format.MessageToJson(config, indent=2),
+        )
```

Comment:
is this different than the log message `self.assertEDSConfigExists(config)` call below?